core/shared/src/main/scala/zio/Config.scala

```diff
@@ -201,10 +201,16 @@ object Config {
     }
   }
   case object Duration extends Primitive[zio.Duration] {
-    final def parse(text: String): Either[Config.Error, zio.Duration] = try Right(java.time.Duration.parse(text))
-    catch {
-      case NonFatal(e) => Left(Config.Error.InvalidData(Chunk.empty, s"Expected a duration value, but found ${text}"))
-    }
+    final def parse(text: String): Either[Config.Error, zio.Duration] =
+      try {
+        try {
+          Right(java.time.Duration.parse(text))
+        } catch {
+          case NonFatal(_) => Right(zio.Duration.fromScala(scala.concurrent.duration.Duration(text)))
```

Comment:
This could be more specific, `DateTimeParseException`

Reply:
Since the `test` parameter is strictly evaluated I don't think this can throw any non-fatal exceptions other than `DateTimeParseException` but could make it more specific to clarify intent.